kratos/utilities/auxiliar_model_part_utilities.h

```diff
@@ -273,6 +283,246 @@ class KRATOS_API(KRATOS_CORE) AuxiliarModelPartUtilities
      */
     void RemoveConditionsAndBelongingsFromAllLevels(Flags IdentifierFlag = TO_ERASE);
 
+
+    /// To Export a Scalar data (Double/int/...)
+    template<typename TDataType>
+    std::vector<TDataType> GetScalarData(
```

Comment:
```suggestion
    void GetScalarData(
```

looking at the changes in the CoSimApp, can you please change the signature of the function to match the one of the `SetValue` fct?
This way we won't have to reallocate memory constantly

Reply:
Yes, I have done this. now the signature of Get... and Set... looks similar. Which uses the DataBuffers to avoid the reallocation of memory. 

I will push the changes now.